bin/setup

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env ruby
+require "pathname"
+require "rake"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"] || "development"
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
+
+def cat(source, target)
```

Comment:
Right, but `cat` is a specific system command that prints to stdout. You're also redirecting `stdout` to a target file, and making checks to see what would change, if anything, so it's not really a pure `cat` function.